src/Kaba.ts

```diff
@@ -77,6 +77,7 @@ export class Kaba
                 h: ["preact", "h"],
                 Fragment: ["preact", "Fragment"],
             }),
+            new CleanWebpackPlugin(),
```

Comment:
You should inline all of these below.
Especially the `CleanWebpackPlugin()` **has to be a different instance** per entry, otherwise you will have quite obscure bugs.